backend/src/main/java/ca/bc/gov/app/controller/client/ClientController.java

```diff
@@ -45,7 +51,36 @@ public Mono<ClientDetailsDto> getClientDetails(
             JwtPrincipalUtil.getProvider(principal)
         );
   }
-
+  
+  @GetMapping("/search")
+  public Flux<ClientListDto> fullSearch(
+      @RequestParam(required = false, defaultValue = "0") int page,
+      @RequestParam(required = false, defaultValue = "10") int size,
+      @RequestParam(required = false, defaultValue = "") String keyword,
+      ServerHttpResponse serverResponse) {
+    log.info("Listing clients: page={}, size={}, keyword={}", page, size, keyword);
+    return clientLegacyService
+        .search(
+            page,
+            size,
+            keyword
+        )
+        .doOnNext(dto -> serverResponse
+            .getHeaders()
+            .putIfAbsent(
+                ApplicationConstant.X_TOTAL_COUNT,
+                List.of(dto.count().toString())
```

Comment:
I would recommend removing the count here and using a Pair instead. This is something I have to fix as well on the forest client api

Reply:
Sure, I will work on this